### ISSUE TITLE
[Feature Request] More Typical Autoroll Damage For Cast Spells That Are Not Attacks

### ISSUE THREAD

**Issue by OutlandishVillainy** [xdy/xdy-pf2e-workbench#1376]
![image](https://github.com/xdy/xdy-pf2e-workbench/assets/93902659/ea470bbe-cf2c-48f9-87a7-55a253720b7f)
This option is misleading, as it stands what it actually does it autoroll damage when you press the save button. This isn't how I think it should work for multiple reasons.

**As a DM**
My players will be the ones clicking the save button, this means my players are clicking the button to roll the damage on themselves. Tradition dictates that they roll, and then I roll the damage, especially because if they critically succeed a basic save and take nothing, I don't want them to know what the damage _could_ have been.

**As a Player**
When a player casts a spell against me, if the damage is rolled when I click the save button, I'm now the one rolling the damage instead of the player. Tradition dictates that players roll their own damage as much as possible, because it's theirs, and having the DM roll it, especially if they roll low, is going to feel bad. The button that rolls damage should be one clicked by the player.

**PF2e Toolbelt Interaction**
This is more of a selfish one, but with toolbelt allowing you to click a button from the damage roll card to visually see the degree of success, I like just telling players to click the damage button as soon as they cast a save spell so I can get access to those, which would be easier if workbench was autorolling the damage anyway. If I'm clicking the save button to get damage rolls, then I can't click the rolls on the damage card because then I'd be rolling twice.
![image](https://github.com/xdy/xdy-pf2e-workbench/assets/93902659/9cec75b6-e2ff-4fc1-ba17-c0ab8c9eb6a0)

**My Request**
Change the option so that it autorolls damage when you cast the spell instead of when you click the save, but also make an option to only do this for players, since as a DM, I don't want to show my players a damage roll if I don't have to, and especially don't want to have them see it before they make the save (tradition dictates they make the save first, then you let them know how screwed they are because of it)

**Comment by Eligarf** [comment]
I agree with the previous post. I couldn't figure out what this feature was doing - I thought it was broken until I read this issue.
